ament_tools/helper.py

```diff
@@ -222,6 +222,10 @@ def extract_argument_group(args, delimiting_option):
     return trimmed_args, extracted_args
 
 
+def compute_deploy_destination(context, filename, dst_subfolder=''):
+    return os.path.join(context.install_space, dst_subfolder, filename)
```

Comment:
Well how the path is constructed could change. I saw non-trivial construction of a value, being duplicated into two places and feared that someday someone might change one but not the other, so I made it into a function.